scripts/reana_bench.py

```diff
@@ -52,7 +55,7 @@ def cli():
 
         \b
         $ cd reana-demo-root6-roofit  # find an example of REANA workflow
-        $ reana_bench.py launch -w roofit50yadage -n 50 -f reana-yadage.yaml  # submit and start
+        $ reana_bench.py launch -w roofit50yadage -n 1-50 -f reana-yadage.yaml  # submit and start
```

Comment:
Shall we keep starting numbers from zero?  If so, please change to `0-49`. Otherwise let's keep `1-50`, it may actually be more user-friendly.

Reply:
I have no preferences. Before it was starting from zero (without justification, just happened like that). It can start from zero or from one, ranges allow that it is up to the user of a script. The question here is what we want to put as an example in docs. I would leave it as it is.

Reply:
If `collect` and `analyze` are beginning-number-agnostic, and are happy to take any numbers (even with holes), then we can keep 1 here.